tests/backends/test_dbscript.py

```diff
@@ -578,3 +578,60 @@ def test_db_script_errors(fileutils, wlmutils, mlutils):
     # an in-memory script
     with pytest.raises(SSUnsupportedError):
         colo_ensemble.add_model(colo_model)
+
+def test_inconsistent_params_add_script(fileutils, wlmutils, mlutils):
```

Comment:
It looks like we're testing a _lot_ of extra code here. Aren't the new validators in a constructor?

Can we build tests around the smallest unit?

```
... 
# you could vary x, y, z and exercise the validation code 
# more directly instead of allowing other code to potentially break
with pytest.raises(SSUnsupportedError):
  dbscript = DBScript(x,y,z)

```

Reply:
Good point especially because the `add_model`, `add_script`, `add_function` are essentially just factory methods for their respective `DBEntity`